core-services/prow/02_config/openshift/machine-os-images/_prowconfig.yaml

```diff
@@ -1,8 +1,8 @@
 tide:
   queries:
   - includedBranches:
-    - openshift-4.10
-    - release-4.10
+    - openshift-4.11
+    - release-4.11
```

Comment:
Looks like we'll need to introduce the GA'd release query in this file (one that requires cherry-pick-approved, backport-risk-assessed) and also add 4.11 branches to the complement query

Reply:
This still needs to be addressed